Modal updates (WIP) (#19504)
MetaMask/metamask-extension: ui/components/component-library/modal-footer/README.mdx

import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';

import { ModalFooter } from './modal-footer';

# ModalFooter

> `ModalFooter` is a container component that handles the scrolling of the `Modal` content

Author: Update to
```suggestion
`ModalFooter` is used as the footer component inside of the `Modal`
```
